spec/rspec/mocks/matchers/receive_spec.rb

```diff
@@ -298,11 +298,26 @@ def receiver.method_missing(*a); end # a poor man's stub...
           let(:receiver) { double }
           let(:wrapped)  { expect(receiver) }
 
-          it 'sets up a message expectation that formats argument matchers correctly' do
-            wrapped.to receive(:foo).with an_instance_of Fixnum
-            expect { verify_all }.to(
-              raise_error(/expected: 1 time with arguments: \(an instance of Fixnum\)\n\s+received: 0 times$/)
-            )
+          context "when a message is not received" do
+            it 'sets up a message expectation that formats argument matchers correctly' do
+              wrapped.to receive(:foo).with an_instance_of Fixnum
+              expect { verify_all }.to(
+                raise_error(/expected: 1 time with arguments: \(an instance of Fixnum\)\n\s+received: 0 times$/)
+              )
+            end
+          end
+
+          context "when a message is received the wrong number of times" do
+            it "sets up a message expectation that formats argument matchers correctly" do
+              wrapped.to receive(:foo).with(anything, hash_including(bar: anything))
+
+              receiver.foo(1, bar: 2)
+              receiver.foo(1, bar: 3)
```

Comment:
@JonRowe just updated the hash syntax. after fixing I was unable to reproduce the failures I was getting before in the cucumber suite (i reverted one of the lines but was still green.) maybe just a quirk on my machine. let me know if there's anything that remains to be done. thanks!